Xpense/urls.py

```diff
@@ -20,5 +20,4 @@
     path('admin/', admin.site.urls),
     path('tips/', include('tips.urls')),
     path('', include('house.urls')),
-
```

Comment:
redundant line?

Reply:
I don't see what you are talking about :( 
![image](https://user-images.githubusercontent.com/93659937/165339410-fcef1d53-cb5f-4033-9136-b34c7b6bea63.png)